### PR TITLE
[TAS-417] ✨ Allow viewing content directly for NFT

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: src/constant/index.js [hunk 1]
```diff
@@ -150,6 +150,11 @@ export const LIKECOIN_NFT_ABOUT_ITEMS = IS_TESTNET
       'likenft1yhsps5l8tmeuy9y7k0rjpx97cl67cjkjnzkycecw5xrvjjp6c5yqz0ttmc',
     ];
 
+// 1. Set undefined to disable view content directly for all NFT classes
+// 2. Set empty array to enable view content directly for all NFT classes
+// 3. Set array of NFT class Id to disable view content directly for specific NFT classes
+export const LIKECOIN_NFT_DIRECT_VIEW_CONTENT = IS_TESTNET ? [] : undefined;
```
[the remark this answers is not in the document]
Comment (nwingt) [reply src/constant/index.js]: Updated

**Comment by nwingt** [comment]
> oh great. is it for wNFT, or epub?

@ckxpress For NFT Book only

image

File: src/constant/index.js [hunk 2]
```diff
@@ -150,10 +150,10 @@ export const LIKECOIN_NFT_ABOUT_ITEMS = IS_TESTNET
       'likenft1yhsps5l8tmeuy9y7k0rjpx97cl67cjkjnzkycecw5xrvjjp6c5yqz0ttmc',
     ];
 
-// 1. Set undefined to disable view content directly for all NFT classes
-// 2. Set empty array to enable view content directly for all NFT classes
+// 1. Set null to enable view content directly for all NFT classes
+// 2. Set empty array to disable view content directly for all NFT classes
 // 3. Set array of NFT class Id to disable view content directly for specific NFT classes
-export const LIKECOIN_NFT_DIRECT_VIEW_CONTENT = IS_TESTNET ? [] : undefined;
+export const LIKECOIN_NFT_DIRECT_ACCESS_ITEMS = IS_TESTNET ? [] : null;
```
Comment (williamchong) [inline src/constant/index.js:156]: do we want to flip this? now it is enable nothing for testnet and all for production

Comment (nwingt) [reply src/constant/index.js]: My mistake... updated

**Review by williamchong — APPROVED** [review]

**Pull request merged**